src/wasm/decoder.cc

```diff
@@ -156,6 +161,10 @@ class LR_WasmDecoder : public Decoder {
       }
     }
     
+    if (FLAG_trace_wasm_decode_time && ok()) {
```

Comment:
You can combine this if with the following if, nesting the decode tracing inside.